### ISSUE TITLE
[Feature Request] Severity

### ISSUE THREAD

**Issue by Liero** [Blazored/FluentValidation#75]
**Is your feature request related to a problem? Please describe.**
FluentValidator supports [Severity](https://docs.fluentvalidation.net/en/latest/severity.html) levels for validation messages.

I only want to prevent form submit, when there are ValidationFailures with severity Error and ignore Warnings and Infos.
Currently, the FluentValidator generates errors for all severity levels. I would like to set minimum level to Error and ignore Warning and Info.

**Describe the solution you'd like**

    <FluentValidator ValidationFailureFilter="@(error => error.Severity >= Severity.Error)" >

**Describe alternatives you've considered**

    <FluentValidator Severity="Severity.Error" >

**Additional context**

Even when I want to enable submit when there are warnings, I still want to be able to show the warnings in the ValidationSummary. 

That might be tricky, but in worst case the FluentValidator might expose Validate, resp ValidateAsync method.

```csharp
FluentValidator _fluentValidatorRef;

async Task OnSubmit()
{
  var result = await _fluentValidatorRef.ValidateAsync();
  bool isValid = !result.Errors.Any(e => e.Severity >= Severity.Error);
  if (isValid) 
  {
      .....
  }
}

```
Alternatively, FluentValidator might expose ValidationResult, but it will propagate to EditContext only those errors, that matches the ValidationFailureFilter. We will be able to catch the Warnings and show them somewhere, but I prefer the custom OnSubmit logic

**Comment by chrissainty** [comment]
Hi @Liero, thanks for raising this. I'm really swamped at the moment but I'm planning on looking over this proposal on Friday (16th). I've not worked with severity levels in FluentValidation before so I just need to get up to speed on it.

**Comment by Liero** [comment]
Once we agree on design proposal, I can post PR.

Concept of Severity in FV is very straightforward -  Rules can be configured with Severity level and validation results will get the Severity flag.

**Comment by chrissainty** [comment]
Hi @Liero,

I've been looking into your request a little and I think we can definitely do this. I haven't had the time I was expecting to go into the details of what a solution would look like. So there are two options. You can suggest something which we can work from. Or you can wait until I get some time, can't tell you when--life is crazy right now.

**Comment by chrissainty** [comment]
@luetm I think this may have been closed in error. I'd be happy to take a PR for the work if you're willing.

**Comment by luetm** [comment]
@chrissainty https://github.com/Blazored/FluentValidation/pull/212 
I took this route of making the whole result available so, if anything besides severity is needed, we don't need to add anything else. Let me know in the PR if the code is acceptable!
